debezium-server/debezium-server-ybexporter/src/main/java/io/debezium/server/ybexporter/EventQueue.java

```diff
@@ -165,3 +180,22 @@ public void sync() {
         }
     }
 }
+
+class SequenceNumberGenerator{
```

Comment:
I would probably directly use an int variable for the same purpose. :-)

Reply:
Fair, but i guess this is java, everything has to be a class :')